app/assets/stylesheets/organizations.scss

```
@@ -0,0 +1,281 @@
+  &:hover {
+    transform: translateY(-3px);
+    box-shadow: 0 8px 20px rgba(66, 185, 131, 0.18);
+    text-decoration: none;
+    color: #1f2937;
+  }
+}
+.org-dash-group-name {
+  font-size: 17px;
+  font-weight: 600;
+  color: #065f46;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)